bigchaindb/log/configs.py

```diff
@@ -0,0 +1,56 @@
+import logging
+
+
+PUBLISHER_LOGGING_CONFIG = {
+    'version': 1,
+    'disable_existing_loggers': False,
+    'root': {
+        'level': logging.DEBUG,
+    },
+}
+
+SUBSCRIBER_LOGGING_CONFIG = {
+    'version': 1,
+    'disable_existing_loggers': False,
+    'formatters': {
+        'console': {
+            'class': 'logging.Formatter',
+            'format': (
+                '%(name)-15s %(levelname)-8s %(processName)-10s %(message)s'
+            ),
+            'datefmt': '%Y-%m-%d %H:%M:%S',
+        },
+        'file': {
+            'class': 'logging.Formatter',
+            'format': ('[%(asctime)s] [%(levelname)s] (%(name)s) '
+                       '%(message)s (%(processName)-10s - pid: %(process)d)'),
+            'datefmt': '%Y-%m-%d %H:%M:%S',
+        },
+    },
+    'handlers': {
+        'console': {
+            'class': 'logging.StreamHandler',
+            'formatter': 'console',
+            'level': logging.INFO,
+        },
+        'file': {
+            'class': 'logging.FileHandler',
+            'filename': '/var/log/bigchaindb.log',
```

Comment:
`$HOME` for now.
In a different pull request we can consider creating `$HOME/.bigchaindb/` folder and store there both the configuration and the logs.